### DISCUSSION TITLE
execution of embedded linux image failed: load error

### DISCUSSION THREAD

**Discussion by aep** [zbm-dev/zfsbootmenu#601] [Questions] [answered]
on EFI load i just get "execution of embedded linux image failed: load error" 

has anyone seen this? i'm reasonably sure this is some sort of BIOS or hardware issue. it works fine when loading the exact same file from nvme, but breaks from sata.  but not sure what that even means.

**Comment by zdykstra** [comment, marked as answer]
I've come across that issue on older Dell (R710) server hardware. You can try rEFInd to load the kernel/initramfs pair. That's worked reliably for me so far.
